packages/lumx-react/src/components/list/ListItem.tsx

```diff
@@ -115,15 +119,18 @@ export const ListItem: Comp<ListItemProps, HTMLLIElement> = forwardRef((props, r
                         linkAs,
                         tabIndex: 0,
                         role: onItemSelected ? 'button' : undefined,
+                        'aria-disabled': isDisabled,
                         ...linkProps,
+                        href: isDisabled ? undefined : linkProps.href,
```

Comment:
Nice. 
I wasn't sure about something and found those two articles: 
https://www.scottohara.me/blog/r2021/05/28/disabled-links.html
https://a11y-guidelines.orange.com/en/articles/disable-elements/

I initially thought you couldn't set a "aria-disabled" on a link and having an empty href was enough, but it looks like it's fine **as long** as you also add `role="link"` (but only when it's a link and it's disabled).